build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -137,20 +137,33 @@ String _entryPointJs(String bootstrapModuleName) => '''
 (function() {
   $_currentDirectoryScript
   $_baseUrlScript
-  var el;
-  el = document.createElement("script");
-  el.defer = true;
-  el.async = false;
-  el.src =
-    baseUrl + "packages/\$sdk/dev_compiler/web/dart_stack_trace_mapper.js";
-  document.head.appendChild(el);
-  el = document.createElement("script");
-  el.defer = true;
-  el.async = false;
-  el.src = baseUrl + "packages/\$sdk/dev_compiler/amd/require.js";
-  el.setAttribute("data-main", _currentDirectory + "$bootstrapModuleName");
-  document.head.appendChild(el);
-})();
+
+  var mapperUri = baseUrl + "packages/\$sdk/dev_compiler/web/dart_stack_trace_mapper.js";
+  var requireUri = baseUrl + "packages/\$sdk/dev_compiler/amd/require.js";
+  var mainUri = _currentDirectory + "$bootstrapModuleName";
+
+  if (typeof document != 'undefined') {
```

Comment:
Or we could be doing something like this - https://stackoverflow.com/a/18002694/39827